COPYRIGHT.txt

```diff
@@ -1939,3 +1939,8 @@ License: Zlib
  2. Altered source versions must be plainly marked as such, and must not be
     misrepresented as being the original software.
  3. This notice may not be removed or altered from any source distribution.
+
+Files: ./thirdparty/nvapi/nvapi_minimal.h
+Comment: Stripped down version of "nvapi.h" from the NVIDIA NVAPI SDK
+Copyright: 2019-2022, NVIDIA CORPORATION. All rights reserved.
+License: MIT
```

Comment:
This should be up between msdfgen and oidn as it is in the thirdparty readme. Other than that, this PR seems ready to merge!